isis/src/control/objs/BundleAdjust/BundleAdjust.cpp

```diff
@@ -3062,9 +3062,13 @@ namespace Isis {
     std::ostringstream ostr;
     ostr << summaryGroup << std::endl;
     m_iterationSummary += QString::fromStdString( ostr.str() );
-    if (m_printSummary) {
+    
+    if (m_printSummary && iApp != NULL) {
       Application::Log(summaryGroup);
     }
+    else {
+      std::cout << summaryGroup << std::endl;
```

Comment:
I'm not sure how I feel about always printing out to std::out. This will only be hit during testing right now, so it's probably fine.